hugegraph-test/src/test/java/org/apache/hugegraph/benchmark/map/MapRandomGetPutThroughputTest.java

```diff
@@ -0,0 +1,126 @@
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.results.format.ResultFormatType;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@BenchmarkMode({Mode.Throughput})
+@Warmup(iterations = 2, time = 1000, timeUnit = TimeUnit.MILLISECONDS)
+@Measurement(iterations = 6, time = 1000, timeUnit = TimeUnit.MILLISECONDS)
+@State(Scope.Benchmark)
+@Fork(3)
+public class MapRandomGetPutThroughputTest {
+
+    private static final int MAP_CAPACITY = 100000;
+    private final ConcurrentHashMap<Integer, Integer> concurrentHashMapNonCap =
+        new ConcurrentHashMap<>();
+
+    private final ConcurrentHashMap<Integer, Integer> concurrentHashMap =
+        new ConcurrentHashMap<>(MAP_CAPACITY);
+
+    private final IntMap.IntMapBySegments intMapBySegments =
+        new IntMap.IntMapBySegments(MAP_CAPACITY);
+
+    private final IntMap.IntMapByEcSegment intMapByEcSegments =
+        new IntMap.IntMapByEcSegment(IntSet.CPUS * 100);
+
+    private static final int THREAD_COUNT = 8;
+
+    /**
+     * The instantiated @State annotation only supports public classes.
+     */
+    @State(Scope.Thread)
+    public static class ThreadState {
+        private final SimpleRandom random = new SimpleRandom();
+
+        int next() {
+            return random.next();
+        }
+    }
+
+    @Benchmark
+    @Threads(THREAD_COUNT)
+    public void randomGetPutByConcurrentHashMapWithNoneInitCap(ThreadState state) {
+        int key = state.next() & (MAP_CAPACITY - 1);
+        if (!concurrentHashMapNonCap.containsKey(key)) {
+            concurrentHashMapNonCap.put(key, state.next());
+        }
+        concurrentHashMapNonCap.get(key);
+    }
+
+    @Benchmark
+    @Threads(THREAD_COUNT)
+    public void randomGetPutByConcurrentHashMapWithInitCap(ThreadState state) {
+        int key = state.next() & (MAP_CAPACITY - 1);
+        if (!concurrentHashMap.containsKey(key)) {
+            concurrentHashMap.put(key, state.next());
+        }
+        concurrentHashMap.get(key);
+    }
+
+    @Benchmark
+    @Threads(THREAD_COUNT)
+    public void randomGetPutByIntMapBySegments(ThreadState state) {
+        int key = state.next() & (MAP_CAPACITY - 1);
+        if (!intMapBySegments.containsKey(key)) {
+            intMapBySegments.put(key, state.next());
+        }
+        intMapBySegments.get(key);
+    }
+
+    @Benchmark
+    @Threads(THREAD_COUNT)
+    public void randomGetPutByIntMapByEcSegment(ThreadState state) {
```

Comment:
to avoid 2 "By", can we rename  randomGetPutByXx to randomGetPutOfXx or randomGetPutForXx:
randomGetPutByIntMapByEcSegment ->  randomGetPutOfIntMapByEcSegment